app/main/views/suppliers.py

```diff
@@ -430,3 +435,31 @@ def update_supplier_framework_details(supplier_id, framework_slug):
         return jsonify(message="Database Error: {0}".format(e)), 400
 
     return jsonify(frameworkInterest=interest_record.serialize()), 200
+
+
+@main.route("/suppliers/<supplier_id>/briefs/<brief_id>", methods=["HEAD"])
```

Comment:
How would we end an endpoint to get a full list of suppliers for a brief, which we might need for eg notifications? The URL also looks a bit strange to me, as briefs don't really belong to suppliers the way it seems to suggest. What do you thing about `/briefs/<brief_id>/suppliers?supplier_id=<id>`? We could then extend it to return the full list of eligible suppliers when called without `supplier_id`.